libs/polysemy-wire-zoo/src/Wire/Sem/Now.hs

```diff
@@ -27,10 +27,10 @@ where
 import Imports
 import Polysemy
 import Polysemy.Check (deriveGenericK)
-import qualified SAML2.WebSSO as SAML
+import Wire.Sem.FromUTC
 
 data Now m a where
-  Get :: Now m SAML.Time
+  Get :: FromUTC a => Now m a
```

Comment:
I don't love this change; is there a reason to prefer this over calling `fromUtcTime` at the call site?

Reply:
My goal here was to make the `Get` action work not only for the `Time` type from that SAML module, but also for `UTCTime`.

If I remove this constraint then I don't know how to make it work. In the interpreter I just can't get types line up. Sure, `fromUTCTime` has to be called, but then there needs to be an instance of `FromUTC` for the type variable `x`, and `x` is completely unconstrained there and I don't see how to make it have this constraint.

I'm open to suggestions, though!

Reply:
i'd suggest `Get :: Now a UtcTime` and then change client code to turn that into the right type

Reply:
Is what I did in commit https://github.com/wireapp/wire-server/pull/2292/commits/c6079896c1db25b741d737a743a194d0fe6e7c72 what you had in mind?